src/generators/web/ui/hooks/useOrama.mjs

```diff
@@ -13,6 +13,12 @@ export default () => {
       schema: {},
     });
 
+    // TODO(@avivkeller): Ask Orama to support this functionality natively
+    /**
+     * @param {any} options
+     */
+    db.search = options => search(db, options);
+
```

Comment:
cc @micheleriva: `@orama/ui` calls `client.search` (https://github.com/oramasearch/orama-ui/blob/main/packages/ui/src/hooks/useSearch.tsx#L94), however, the non-Cloud client don't implement this function. Instead, they use `search(client, ...)` syntax.

Is there a way for Orama to add `client.search()` functionality to the non-Cloud clients, **or** support `search(client, ...)` in the UI library?